Sources/SearchParameters.swift

```diff
@@ -758,6 +758,43 @@ import Foundation
             }
         }
     }
+    
+    /// Remove a refinement for a given refinement condition
+    ///
+    /// - parameter condition: the condition to evaluate
+    ///
+    @objc public func removeNumericRefinements(where condition: (NumericRefinement) -> Bool) {
+        let oldRefinements = numericRefinements
+        oldRefinements.forEach { (name: String, refinements: [NumericRefinement]) in
+            let newRefinements = refinements.filter({ !condition($0) })
+            numericRefinements[name] = newRefinements.isEmpty ? nil : newRefinements
+        }
+    }
+    
+    /// Updates a refinement for a given metric
```

Comment:
By convention, I try to use the infinitive (rather than the 3rd person) to describe functions. Also, a trailing dot would be appreciated, because some tools treat the first sentence as a "short description".